odoo/tests/test-uninstalls.py

```diff
@@ -107,6 +112,25 @@ def test_uninstall(args):
         install(args.database, module_id, module_name)
 
 
+def test_scripts(args):
+    """ Tries to launch standalone scripts tagged with @post_testing """
+    # load the registry once for script discovery
+    odoo.registry(args.database)
```

Comment:
To avoid explicit imports of `tests` in modules, the Python tests are discovered with the function `get_test_modules`:
```suggestion
    registry = odoo.registry(args.database)
    for module_name in registry._init_modules:
        # this imports odoo.addons.<module_name>.tests
        odoo.modules.module.get_test_modules(module_name)
```

Reply:
good to know